src/seedu/addressbook/AddressBook.java

```diff
@@ -1144,14 +1184,14 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes prefix(p/, d/, etc) from parameter string
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
-     * @return  string without the sign
+     * @param fullString  original string with prefix
+     * @param sign  the prefix to be removed
+     * @return  string without the prefix
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
+        return fullString.replace(prefix, "");
```

Comment:
this solution still has the same problem as earlier. you need to remove only the prefix;
in the current implementation, you will replace every occurrence of `prefix` in the full string with null.